policyengine_us/variables/gov/states/me/tax/income/credits/me_child_care_credit/me_step4_share_of_child_care_expenses.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class me_step4_share_of_child_care_expenses(Variable):
```

Comment:
For consistency with the parameter
```suggestion
class me_step_4_share_of_child_care_expenses(Variable):
```